gradient/api_sdk/repositories/models.py

```diff
@@ -77,13 +78,26 @@ def _get_request_json(self, instance_dict):
 
     def _get_request_files(self, file_handle):
         """
-        :param file file_handle:
+        :param str file_handle: path to Model that will be uploaded
         """
         if not file_handle:
             return None
 
-        file_name = os.path.basename(file_handle.name)
-        return [(file_name, file_handle)]
+        return self._prepare_files(file_handle)
```

Comment:
Shouldn't file_handle be changed to path?

Reply:
then I should change the 'CreateResource' class implementation or `_send_create_request` method implementation and all calls of it. So I decided that for this one call I would leave it as it is